streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -100,7 +102,7 @@ public ProcessorStateManager(final TaskId taskId,
         try {
             baseDir = stateDirectory.directoryForTask(taskId);
         } catch (final ProcessorStateException e) {
-            throw new LockException(String.format("%s Failed to get the directory for task %s. Exception %s",
+            throw new LockException(String.format("%sFailed to get the directory for task %s. Exception %s",
```

Comment:
If "Exception %s" used, please use `e.getMessage`. Otherwise to throw exception, we don't need `Exception %s`

Reply:
because it was there before my changes so preferring to leave it as.